OpenRA-OpenAL-CS.nuspec

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>OpenRA-OpenAL-CS</id>
+    <version>0.0.1</version> <!-- Version is set during packaging. -->
+    <authors>OpenRA</authors>
+    <owners>OpenRA</owners>
+    <projectUrl>https://github.com/OpenRA/OpenAL-CS</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>OpenAL-CS modified for OpenRA</description>
+    <copyright>Copyright (c) The OpenRA Developers and Contributors</copyright>
```

Comment:
Does this need to include the original copyright?

Reply:
Hopefully @Mailaender can answer that better than I could.

Reply:
> Copyright (c) 2014-2015 Ethan Lee

Reply:
and probably

> Creative Labs

as you are bundling the native libraries. Ideally, they should be in a separated NuGet as they have a different license.